independent-publisher-connectors/HubSpot CRM/apiProperties.json

```diff
@@ -0,0 +1,75 @@
+{

```

Comment:
@troystaylor Did you tested this connector in your dev/local enviornment, If yes, didn't you get issues ?

Reply:
Copied properties from CMS connector, fixed

Reply:
Updated